### ISSUE TITLE
Update user data templates with the chef user

### ISSUE THREAD

**Issue by scarolan** [chef-cft/bjc#470]
I've only got one data point so far, but `kitchen converge` is failing in bjc-ecommerce on a fresh build from master.  

```
       [2017-03-25T19:55:54+00:00] ERROR: Error connecting to https://automate.automate-demo.com/data-collector/v0/, retry 1/5
       [2017-03-25T19:55:59+00:00] ERROR: Error connecting to https://automate.automate-demo.com/data-collector/v0/, retry 2/5
       [2017-03-25T19:56:04+00:00] ERROR: Error connecting to https://automate.automate-demo.com/data-collector/v0/, retry 3/5
```

**Comment by ChefRycar** [comment]
You are correct sir. noticed this yesterday. The reason is that kitchen configures it with a user_data file to spoof a host entry with the proper IP for the automate server. the switch from Administrator to chef means it can't find it anymore and I'll need to update those paths in the template. 

Also, think I'm going to stop having that raise an error. Unless someone has strong differing feelings, my preference would be for failed data collection to still result in a failed converge, with the option to raise an error commented out so it can be used for troubleshooting when necessary.

**Comment by ChefRycar** [comment]
FYI: Quick fix in the meantime: Update kitchen.yml to point at C:\Users\chef\*_user_data instead of C:\Users\Administrator

**Comment by ChefRycar** [comment]
Moving to ready for promoting 2.x to stable. 
tl;dr of what's needed:
- in the bjc_workstation cookbook, update all test kitchen templates/files that are presently looking for user_data in C:\Users\Administrator to C:\Users\chef instead.
- BONUS: There is a #{home} variable being set in a function already that would be preferable to use rather than hardcoding. That may require a re-factor however, as certain things are set in cookbook_files presently.
